diff --git a/dist/index.js b/dist/index.js
index 7af9fca6297b30f9a59f77dc5330200a37f0c432..904cfcfaade5eeb5e6b55e7d6e097087e474cf9e 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -23,7 +23,7 @@ function updateBundledDependencies(root, projectName, configurationName, node, d
         overrides: {},
         target: {
             project: projectName,
-            target: 'build',
+            target: 'build:es5',
             configuration: configurationName,
         }
     }, node);
@@ -72,7 +72,7 @@ function toBundableDependency(context, projectGraph, dependencyName) {
         overrides: {},
         target: {
             project: context.projectName,
-            target: 'build',
+            target: 'build:es5',
             configuration: context.configurationName
         }
     }, projectGraph.nodes[dependencyName]);
