diff --git a/src/Assets.js b/src/Assets.js
index a94d722574aab457395814e858fcf36741714feb..434023fd8fe8dbe3cbce39ce02a9501999cbcc5e 100644
--- a/src/Assets.js
+++ b/src/Assets.js
@@ -3,7 +3,11 @@
 const AssetPaths = require("./node-haste/lib/AssetPaths");
 const crypto = require("crypto");
 const fs = require("fs");
-const getImageSize = require("image-size");
+const _imageSizeModule = require("image-size");
+const getImageSize =
+  typeof _imageSizeModule === "function"
+    ? _imageSizeModule
+    : _imageSizeModule.imageSize || _imageSizeModule.default;
 const path = require("path");
 function isAssetTypeAnImage(type) {
   return (
