diff --git a/src/utils/package-json/update-package-json.js b/src/utils/package-json/update-package-json.js
index 1190b046c4b3668afe1b824d48ee699689d2498e..6197b22df15c6f80dae5776e9a5e1d9dc4b77212 100644
--- a/src/utils/package-json/update-package-json.js
+++ b/src/utils/package-json/update-package-json.js
@@ -72,7 +72,11 @@ function addMissingDependencies(packageJson, { projectName, targetName, configur
         }
         else {
             const packageName = entry.name;
+            if (!!workspacePackageJson.devDependencies?.[packageName]) {
+                return;
+            }
             if (!((_f = packageJson.dependencies) === null || _f === void 0 ? void 0 : _f[packageName]) &&
+                !((_f = packageJson.devDependencies) === null || _f === void 0 ? void 0 : _f[packageName]) &&
                 !((_g = packageJson.peerDependencies) === null || _g === void 0 ? void 0 : _g[packageName])) {
                 const outputs = (0, devkit_1.getOutputsForTargetAndConfiguration)({
                     overrides: {},